apps/console/src/features/applications/components/settings/attribute-management/advance-attribute-settings.tsx

```diff
@@ -169,12 +182,39 @@ export const AdvanceAttributeSettings: FunctionComponent<AdvanceAttributeSetting
         return claimURI;
     });
 
+    interface AdvanceAttributeSettingsErrorValidationInterface {
+
+        sectorIdentifierURI?: string;
+    }
```

Comment:
move to the models folder

Reply:
fixed with [a5f0e39](https://github.com/wso2/identity-apps/pull/4284/commits/a5f0e399c3a9fa90a48a937face8fe7d2f554faf)